bundles/org.openhab.binding.bmwconnecteddrive/src/main/resources/OH-INF/thing/destination-channel-types.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="bmwconnecteddrive"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<channel-type id="destination-name-channel">
+		<item-type>String</item-type>
+		<label>Destination Name</label>
+	</channel-type>
+	<channel-type id="destination-latitude-channel">
+		<item-type>Number</item-type>
+		<label>Destination Latitude</label>
+	</channel-type>
+	<channel-type id="destination-longitude-channel">
+		<item-type>Number</item-type>
+		<label>Destination Longitude </label>
+	</channel-type>
+	<channel-type id="destination-size-channel">
+		<item-type>String</item-type>
```

Comment:
Is this supposed to be a number instead?

Reply:
Sure, checked also other size values - they are ok.